tests/Reports/ElementsInUseReportTest.php

```diff
@@ -49,15 +50,8 @@ public function testSourceRecords()
     {
         $records = (new ElementsInUseReport)->sourceRecords();
 
-        $this->assertInstanceOf(ArrayList::class, $records);
-        $this->assertNotContains(BaseElement::class, $records->toArray(), 'BaseElement is excluded');
-
-        $this->assertContainsOnlyInstancesOf(ArrayData::class, $records);
-
-        foreach ($records as $record) {
-            $this->assertNotNull($record->Icon, 'Fields have an icon');
-            $this->assertNotNull($record->Type, 'Fields have a type');
-        }
+        $this->assertInstanceOf(DataList::class, $records);
+        $this->assertContainsOnlyInstancesOf(BaseElement::class, $records);
```

Comment:
Isn't this technically removing the granularity of "The report does NOT contain `BaseElement` (without extension) objects"?

Reply:
Yeah, I should add it back. Thanks